File: src/main/java/it/contrader/dto/MagazzinoDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data

public class MagazzinoDTO {
	
	private Long id;

	private OggettoDTO oggetto;
		
	private int capienza;
		
	private CodiceDTO codice;
		
	private boolean cancellato;
}

File: src/dto/magazzinodto.ts
import { OggettoDTO } from './oggettodto';
import { CodiceDTO } from './codicedto';


export class MagazzinoDTO {

    id: number;

    oggettoId: number;

    oggettoNome: string;

    capienza: number;

    codice: CodiceDTO;

    cancellato: boolean;

}